test/test_operations.py

```diff
@@ -544,6 +552,10 @@ def test(self):
 
 class TestAtenTensorTo(XlaTestCase):
 
+  @unittest.skipIf(
+      os.environ.get('PYTORCH_XLA_TESTS_SKIP'),
+      'To avoid new feature developing, disable failed PyTorch/XLA test on TPUVM'
```

Comment:
@will-cromar FYI, I think a lot of these test failed because of PJRT, since we only use PJRT for TPU CI.